vmware-base/src/main/java/com/cloud/hypervisor/vmware/mo/VirtualMachineMO.java

```diff
@@ -781,6 +782,9 @@ public boolean createFullClone(String cloneName, ManagedObjectReference morFolde
 
         relocSpec.setDatastore(morDs);
         relocSpec.setPool(morResourcePool);
+
+        setDiskProvsioningType(relocSpec, morDs, diskProvisioningType);
+
```

Comment:
when "_diskProvisioningType == null_" here, which provisioning type the disk will fall back to ?  strictness check required here ?

Reply:
Always thin